aspnetcore/client-side/spa/blazor/index.md

```diff
@@ -86,7 +86,7 @@ For apps that require third-party JavaScript libraries and browser APIs, Blazor
 
 ## Code sharing and .NET Standard
 
-Apps can reference and use existing [.NET Standard](/dotnet/standard/net-standard) libraries. .NET Standard is a formal specification of .NET APIs that are common across .NET implementations. Blazor supports .NET Standard 2.0. APIs that aren't applicable inside a web browser (for example, accessing the file system, opening a socket, threading, and other features) throw <xref:System.PlatformNotSupportedException>. .NET Standard 2.0 class libraries can be shared across different .NET platforms, like Blazor, .NET Framework, .NET Core, Xamarin, Mono, and Unity.
+Apps can reference and use existing [.NET Standard](/dotnet/standard/net-standard) libraries. .NET Standard is a formal specification of .NET APIs that are common across .NET implementations. Blazor implements .NET Standard 2.0. APIs that aren't applicable inside a web browser (for example, accessing the file system, opening a socket, threading, and other features) throw <xref:System.PlatformNotSupportedException>. .NET Standard 2.0 class libraries can be shared across different .NET platforms, like Blazor, .NET Framework, .NET Core, Xamarin, Mono, and Unity.
```

Comment:
Thanks @thomasclaudiushuber 

WRT ...

> .NET Standard 2.0 class libraries ...

I still prefer the versionless approach. ".NET Standard" in this context is only here AFAIK to distinguish them from PCLs.

Let's let @danroth27's look and see if that's correct.

We have a hard time on this repo keeping up with explicit versions release-to-release, so we generally prefer *not* to use them whenever possible. 🏃🏃🏃😅

Reply:
I think it's totally fine to omit the version here. I added it, because .NET Standard 2.1 is not supported on .NET Framework. But I agree, one version per paragraph might be enough. :)